### ISSUE TITLE
Users should not be able to save MP3s to disk

### ISSUE THREAD

**Issue by escowles** [pulibrary/figgy#2082]
- [ ] Research streaming (HLS vs. DASH vs. ??) and format support (MP3 vs. AAC vs. ??)
- [ ] Support HLS or DASH instead of allowing users to download access files

Child of #2066

**Comment by tpendragon** [comment]
Asked about this in the UV channel and got the following reply:
```
andyirving [1:22 AM]
you can add a service using the uv context
using the `disableUI` property

and `mediaDownload`
```

**Comment by escowles** [comment]
Easy-and-good-enough: suppress the download button
Harder-and-better: do HTTP Live Streaming (supported by nginx, won't work in dev, so we'd need to fallback to current behavior).

**Comment by hackmastera** [comment]
closed by #2161
